CASSANDRA-17555 April 2022 blog "Inside Cassandra: An Interview with Project Contributor, Aleksandr Sorokoumov" (#124)
apache/cassandra-website: site-content/source/modules/ROOT/pages/blog.adoc

- Replace post tile, date, description and link to you post.
////

//start card
[openblock,card shadow relative test]
----
[openblock,card-header]
------
[discrete]
=== Inside Cassandra: An Interview with Project Contributor, Aleksandr Sorokoumov
[discrete]

> ==== April 14, 2022

Reviewer: ```suggestion
==== April 21, 2022
```

------
[openblock,card-content]
------

> We continue our Inside Cassandra series with a Q&A with Aleksandr Sorokoumov. Like Lorina Poland and others, he recently accepted the committer position in recognition of his contributions, including his work on improving serialization efficiency.

Reviewer: We need to limit the content so it fits in the card:
```suggestion
We continue our Inside Cassandra series with a Q&A with Aleksandr Sorokoumov who recently accepted the committer position in recognition of his contributions.
```
